src/app/components/tooltip/tooltip.ts

```diff
@@ -169,20 +173,27 @@ export class Tooltip implements AfterViewInit, OnDestroy {
     ngAfterViewInit() {
         if (isPlatformBrowser(this.platformId)) {
             this.zone.runOutsideAngular(() => {
-                if (this.getOption('tooltipEvent') === 'hover') {
-                    this.mouseEnterListener = this.onMouseEnter.bind(this);
-                    this.mouseLeaveListener = this.onMouseLeave.bind(this);
-                    this.clickListener = this.onInputClick.bind(this);
-                    this.el.nativeElement.addEventListener('mouseenter', this.mouseEnterListener);
-                    this.el.nativeElement.addEventListener('click', this.clickListener);
-                    this.el.nativeElement.addEventListener('mouseleave', this.mouseLeaveListener);
-                } else if (this.getOption('tooltipEvent') === 'focus') {
-                    this.focusListener = this.onFocus.bind(this);
-                    this.blurListener = this.onBlur.bind(this);
-
-                    let target = this.getTarget(this.el.nativeElement);
-                    target.addEventListener('focus', this.focusListener);
-                    target.addEventListener('blur', this.blurListener);
+                if (this.isMobile()) {
```

Comment:
There are too many nested if, which makes this code hard to read. Can we split it into smaller sub-methods?